eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -447,3 +448,13 @@ stages:
                     - docs-ref-services/
                     - metadata/
                   DocValidationImageId: "$(DocValidationImageId)"
+
+              - template: /eng/pipelines/templates/steps/fetch-package-list.yml
```

Comment:
Since this doesn't impact how docs or ToC are generated yet let's add it to the Docs.MS release too. It'll help to start populating the metadata.